### ISSUE TITLE
Gmod?

### ISSUE THREAD

**Issue by mehdi171** [iCrazyBlaze/Xash3D-Emscripten#9]
Is it possible to have Gmod (like the first versions) on a zip archive that you can play?

**Comment by slambo311** [comment]
No, Garry's mod runs on the Source engine, not Goldsource or Xash3D.

**Comment by mehdi171** [comment]
Well, is it possible to have CS 1.6 ? (yes im aware that play-cs.com exists but i wanna play with NPCs and make servers without having to pay)

**Comment by slambo311** [comment]
You'll have to have the actual Counter-strike files to use it, but it might. Practically guaranteed if you use the actual Xash3D.

**Comment by Samdan9** [comment]
There is GoMod however you need to modify the valve files for base support and then add the modded files and hope that it works. No Gmod since that is a Source 1 game and this is Goldsource.
